mmtrack/models/trackers/base_tracker.py

```diff
@@ -204,8 +204,8 @@ def crop_imgs(self, img, img_metas, bboxes, rescale=False):
         if rescale:
             bboxes[:, :4] *= torch.tensor(img_metas[0]['scale_factor']).to(
                 bboxes.device)
-        bboxes[:, 0::2] = torch.clamp(bboxes[:, 0::2], min=0, max=w)
-        bboxes[:, 1::2] = torch.clamp(bboxes[:, 1::2], min=0, max=h)
+        bboxes[:, 0::2] = torch.clamp(bboxes[:, 0::2], min=0, max=w-1)
+        bboxes[:, 1::2] = torch.clamp(bboxes[:, 1::2], min=0, max=h-1)
```

Comment:
I think it's better to use the following code to fix the bug.
`bboxes[:, 0] = torch.clamp(bboxes[:, 0], min=0, max=w - 1)`
`bboxes[:, 2] = torch.clamp(bboxes[:, 2], min=1, max=w)`